rero_ils/query.py

```diff
@@ -38,20 +38,22 @@
 _PUNCTUATION_REGEX = re.compile(r'[:,\?,\,,\.,;,!,=,-]+(\s+|$)')
 
 
-def and_term_filter(field, must=[], must_not=[]):
```

Comment:
Usually I prefer explicit instead of implicit. But it is up to you.

Reply:
we can't use empty array as function default param because this is "Immutable" (it's you that explain me this fact).
So we need to use `None` as default param signature and write a `must = must or []` into the function body to solve this problem.